common/changes/iconame-reducesize_2017-01-10-01-53.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Icon: -BREAKING CHANGE- Change IconName from an enum to type to greatly reduce file size. Any component that is currently consuming the icon enum will break.",
```

Comment:
I'm still confused on why we're doing this. If it's a breaking change, why not mark it as major? Our releases are still tagged as "pre-release" and we can communicate that v1.0.0 is not to suggest that this is completely production-ready.